server/settings/imageio.py

```diff
@@ -11,15 +11,41 @@
     ColorspaceConfigurationModel,
 )
 
+def nuke_creator_plugins_enum():
+    return [
+        {"value": "CreateWritePrerender", "label": "Prerender (write)"},
+        {"value": "CreateCamera", "label": "Camera (3d)"},
+        {"value": "CreateGizmo", "label": "Gizmo (group)"},
+        {"value": "CreateWriteImage", "label": "Image (write)"},
+        {"value": "CreateModel", "label": "Model (3d)"},
+        {"value": "CreateBackdrop", "label": "Nukenodes (backdrop)"},
+        {"value": "CreateWriteRender", "label": "Render (write)"},
+        {"value": "CreateSource", "label": "Source (read)"},
+    ]
+
+
+def nuke_node_class_enum():
+    return [
+        {"value": "Write", "label": "Write [Image]"},
+        {"value": "Read", "label": "Read [Image]"},
+        {"value": "Group", "label": "Group [Other]"},
+        {"value": "Camera4", "label": "Camera [3D]"},
+        {"value": "Camera2", "label": "Camera [3D Classic]"},
+        {"value": "Scene", "label": "Scene [3D Classic]"},
+        {"value": "BackdropNode", "label": "Backdrop [Other]"}, 
+    ]
```

Comment:
Potentially not for this PR, but I'm wondering if there is a way to make these enums dynamically retrieved ?
We would need to add a class attribute to each creator to get the associated Nuke node Class and gather the creators.

Benefit imo would be that we get server settings follow code adjustments.